packages/nx/src/utils/package-manager.ts

```diff
@@ -113,6 +115,19 @@ export function getPackageManagerCommand(
         list: 'npm ls',
       };
     },
+    bun: () => {
+      return {
+        install: 'bun install',
+        ciInstall: 'bun install --frozen-lockfile',
+        add: 'bun add',
+        addDev: 'bun add -D',
+        rm: 'bun remove',
+        exec: 'bunx',
+        run: (script: string, args: string) => `bun ${script} ${args}`,
+        // list not aviable yet. But cant see wherte its used
+        list: '',
```

Comment:
Isn't this `bun pm ls`?

Reply:
Thank you! I'll  make the change wasn't aware of that command i did go looking for it